packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1719,6 +1719,12 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
       declaration
     )
 
+    const hasInvalidPhoneForContact = Boolean(
+      errorsOnFields.registration?.contactPoint?.nestedFields?.registrationPhone
+        ?.length > 0 ||
+        errorsOnFields.registration?.contactPoint?.errors.length !== 0
```

Comment:
This will probably evaluate to true if the lhs is undefined no? Do we want that?